### PR TITLE
Added rangeSelectorBounds option

### PR DESCRIPTION
**Pull request by neurot1k**
Limits the range selector so that a section of the data is always visible.

### CODE REVIEW

**Comment by danvk** [comment]
Does this control the x-axis range of the range selector? Why would you want to show less than the full domain of the data?

File: auto_tests/tests/range_selector.js [hunk 1]
```diff
@@ -21,11 +21,11 @@ cleanupAfterEach();
 var restoreConsole;
 var logs = {};
 beforeEach(function() {
-  restoreConsole = Util.captureConsole(logs);
+  //restoreConsole = Util.captureConsole(logs);
```
Comment (LM450N) [inline auto_tests/tests/range_selector.js]: Could you remove the useless comments please?

**Comment by neurot1k** [comment]
Yes, it limits the x-axis range. Why would you want this? Perhaps you have some highlighted data that you always want displayed. 

Specifically, I'm using the range selector to create a segment of a larger gps track and annotations to define waypoints. Once a waypoint is defined it must be a part of the segment so I want to limit the range that can be selected. 

I understand this may be a specialized case, so if you don't think it adds any value to the project, no worries.

File: auto_tests/tests/range_selector.js (hunk 1 above)
Comment (neurot1k) [inline auto_tests/tests/range_selector.js]: Thanks, those have been removed.

**Comment by danvk** [comment]
Yeah, I don't see this as a general use case. If you want to make only a portion of your data accessible in the chart, you're better off filtering the data before you give it to dygraphs.

Thanks for your contribution, though!

**Comment by LM450N** [comment]
_[my two cents]_ 💸 
This feature could prevents to reach a white screen when the user move the graph where there is no data?


move-graph-out-range

**Pull request closed without merging**